### ISSUE TITLE
script_weatherers

### ISSUE THREAD

**Issue by bobsurf00** [NOAA-ORR-ERD/PyGnome#60]
Hi,
Looking for a bit of guidance on script_weatherers.  When I run, it doesn't appear to complete for me giving the following warning
![image](https://user-images.githubusercontent.com/45707327/89782388-c6a3b580-db47-11ea-9a04-b4e6d9d27de4.png)
All files attached.
Appreciate your thoughts.
Thanks
Rob
[script_weatherers_uncertain.zip](https://github.com/NOAA-ORR-ERD/PyGnome/files/5050790/script_weatherers_uncertain.zip)

**Comment by ChrisBarker-NOAA** [comment]
It's failing with the KDE for surface concentration. which is because you have no diffusion (RandomMover), so all the elements are on top of each other.

The odd part is that I thought that the KDE for surface concentration was turned off by default, but I guess not. 

Enclosed is a script that works. I also cleaned it up a bit -- we've added a number of handy things in 

the gnome.scripting module that make scripting easier.


[script_weatherers.py.txt](https://github.com/NOAA-ORR-ERD/PyGnome/files/5054321/script_weatherers.py.txt)
